lib/blockchain.go

```diff
@@ -2876,6 +2876,45 @@ func (bc *Blockchain) ValidateTransaction(
 	return nil
 }
 
+// ValidateTransactions creates a UtxoView and sees if the transactions can be connected
+// to it. If a mempool is provided, this function tries to find dependencies of the
+// passed-in transactions in the pool and connect them before trying to connect the
+// passed-in transactions.
+func (bc *Blockchain) ValidateTransactions(
+	txns []*MsgDeSoTxn, blockHeight uint32, verifySignatures bool, mempool Mempool) error {
```

Comment:
nit, see comment below
```suggestion
	txns []*MsgDeSoTxn, blockHeight uint32, verifySignatures bool, mempool Mempool,
) error {
```